sovrin_client/test/agent/conftest.py

```diff
@@ -413,7 +413,7 @@ def chk():
         assert link.remoteIdentifier == inviteeAcceptanceId
         assert link.remoteEndPoint[1] == inviteeAgent.endpoint.ha[1]
 
-    emptyLooper.run(eventually(chk, timeout=10, retryWait=0.2))
+    emptyLooper.run(eventually(chk))
```

Comment:
I reverted this line to the state in which it was previously. The default values of timeout and retryWait arguments of eventually function are used now. I ran all the tests on Windows 10 and all they passed.